calico-enterprise/_includes/components/InstallOpenShift.js

```diff
@@ -57,9 +57,19 @@ export default function InstallOpenShift(props) {
       </p>
       <CodeBlock language='bash'>
         sed -i 's/\(OpenShiftSDN\|OVNKubernetes\)/Calico/' install-config.yaml{'\n'}
-        sed -i 's/platform: {}/platform:\n{'\t'}aws:\n{'\t'}type: m4.xlarge/g' install-config.yaml
       </CodeBlock>
 
+      <Admonition type='note'>
+        <p>By default openshift-installer creates 3 replicas, you can change these settings by modifying the cloud-provider part in the install-config.yaml</p>
+        <p>The following example changes the default deployment instance type and replica quantity.</p>
+        <CodeBlock language='yaml'>
+        platform: 
+          aws:
+            type: m5.xlarge
+        replicas: 2
+        </CodeBlock>
```

Comment:
This is currently being processed without line breaks or indentation. I'd suggest you do something like this:

```suggestion
        <CodeBlock language='yaml' title="install-config.yaml">
        {...
platform: 
  aws:
    type: m5.xlarge
replicas: 2
...
        </CodeBlock>
```
This also adds a title and ellipses to make it clear that this is a snippet from that YAML. 

Would it be easier to see where you are by adding the next level up from `platform`?

Reply:
Easier for user? yes
But it will be more difficult to keep it uptodate for us too.